scripts/verify-no-string-interpolation-leftovers

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+exit_code=0
+for class in $(find . -name '*.class' -type f); do
+  strings_output=$(strings $class)
```

Comment:
`javap` (java "deassembler") would be much more correct here, `strings` is just a nasty heurestics; I tried the former but it was prohibitively slow, esp. given that there's surprisingly no option to just list constant pool; the entire constant pool is only listed in the most compute-intensive `-verbose` mode